### DISCUSSION TITLE
Tree grid (nested complex data)

### DISCUSSION THREAD

**Discussion by tibi-extera** [quasarframework/quasar#14925] [General - Components / Directives / etc]
Hi,

What would be the best way to display hierarchical data something like this:

| name         | Field 1| Field 2|
|--------------|-----------|------------|
| + Parent 1 | value 1:1| value 1:2        |
| &nbsp;&nbsp;&nbsp;&nbsp;+ Child 1.1 | value 1.1:1  | value 1.1:2 |
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Child 1.1.1 | value 1.1.1:1  | value 1.1.1:2 |
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Child 1.1.2 | value 1.1.2:1  | value 1.1.2:2 |
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Child 1.1.3 | value 1.1.3:1  | value 1.1.3:2 |
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Child 1.2 | value 1.2:1  | value 1.2:2 |
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Child 1.3 | value 1.3:1  | value 1.3:2 |
| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Child 1.4 | value 1.4:1  | value 1.4:2 |

**Comment by MilosPaunovic** [comment]
https://quasar.dev/vue-components/tree#introduction
